aggregator/src/binaries/aggregation_job_driver.rs

```diff
@@ -121,6 +123,12 @@ pub struct Config {
     /// will reduce the amount of database contention during leader aggregation, while increasing
     /// the cost of collection.
     pub batch_aggregation_shard_count: u64,
+
+    /// Defines the number of shards to break report & aggregation metric counters into. Increasing
+    /// this value will reduce the amount of database contention during report uploads &
+    /// aggregations, while increasing the cost of getting task metrics.
```

Comment:
We should copy this comment over to `binaries/aggregator.rs`, since that field is now used for two kinds of counters.